src/pe_source/data/sixgill/config.py

```diff
@@ -0,0 +1,45 @@
+"""Get cybersixgill API token."""
+
+# Standard Python Libraries
+from configparser import ConfigParser
+import glob
+import os
+
+# Third-Party Libraries
+import requests
+
+# Configuration
+SECTION = "sixgill"
+BASE_DIR = os.path.abspath(os.path.join(__file__, "../../../.."))
+REPORT_DB_CONFIG = glob.glob(f"{BASE_DIR}/**/*.ini", recursive=True)[0]
+
+
+def token():
+    """Retrieve bearer token from Cybersixgill client."""
+    if os.path.isfile(REPORT_DB_CONFIG):
+        parser = ConfigParser()
+        parser.read(REPORT_DB_CONFIG, encoding="utf-8")
+        if parser.has_section(SECTION):
+            params = parser.items(SECTION)
+            _id, _secret = params[0], params[1]
+            client_id = _id[1]
+            client_secret = _secret[1]
+        else:
+            raise Exception(
+                "Section {} not found in the {} file".format(SECTION, REPORT_DB_CONFIG)
+            )
+    else:
+        raise Exception("Database.ini file not found.")
+    url = "https://api.cybersixgill.com/auth/token/"
+    headers = {
+        "Content-Type": "application/x-www-form-urlencoded",
+        "Cache-Control": "no-cache",
+    }
+    payload = {
+        "grant_type": "client_credentials",
+        "client_id": client_id,
+        "client_secret": client_secret,
+    }
+    resp = requests.post(url, headers=headers, data=payload).json()
+    token = resp["access_token"]
+    return token
```

Comment:
Can we centralize the all db config and associated functions at pe_reports/data/config.py?

Reply:
Ya good call. Addressed [here](https://github.com/cisagov/pe-reports/pull/117/commits/a45d956ff68e982e4a0a700e49bc047e7d8f44ac)

Reply:
@aloftus23 , 
Thanks for making the changes.